circle.yml

```diff
@@ -32,6 +34,7 @@ notify:
 test:
   override:
     - bundle exec rake lint:bash
+    - cd $GO_PROJECT && gometalinter.v1 --deadline=20s cmd/
```

Comment:
Good idea, replaced it with `bin/lint_go`. In the long term I think we can phase out the cumbersome rake tasks in favor of much simpler Bash scripts, to be easier to use and more conforming with the standard defined in https://github.com/originate/o-tools-node.